src/Modules/DisplayTask/components/index.tsx

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+import { NavLink } from "react-router-dom";
+
+import { InfinitySpin } from "react-loader-spinner";
+
+import { ToDoData } from "../../../data/types/types";
+
+interface IProps{
+    data:ToDoData,
+    loading:boolean,
+    error:string,
+    markTodoCompleted:(completed:string)=>void,
+    deleteTodo:(id:number)=>void
+}
+
+const DisplayTaskComponent :React.FC<IProps> = ({data,loading,error,markTodoCompleted,deleteTodo}) => {
+  return (
+    <>
+      {loading ? (
+        <div className="loading-spinner">
```

Comment:
create a separate component for this spinner

Reply:
Done